homework1/cash_register.py

```diff
@@ -0,0 +1,49 @@
+class CashRegister:
+
+    def __init__(self):
+        self.total_items = []
```

Comment:
Nice logic. But don't you think a single dictionary would have served? Where the key is the name of the product and the value is the price. E.g: 
{ "Soap": 2.1, "Bread": 1.87}